ddtrace/tracer/tracer.go

```diff
@@ -117,10 +121,14 @@ func Start(opts ...StartOption) {
 	if t.config.logStartup {
 		logStartup(t)
 	}
+	startAppSec(t)
 }
 
 // Stop stops the started tracer. Subsequent calls are valid but become no-op.
 func Stop() {
+	if appsec := internal.GetGlobalTracer().(*tracer).appsec; appsec != nil {
```

Comment:
You should first check that the global tracer can cast into `*tracer` before accessing any fields on it.
```suggestion
	if v, ok := internal.GetGlobalTracer().(*tracer); ok && v.appsec != nil {
	    // use v.appsec
```